coincurve/__init__.py

```diff
@@ -1,9 +1,44 @@
-from coincurve.context import GLOBAL_CONTEXT, Context
-from coincurve.keys import PrivateKey, PublicKey, PublicKeyXOnly
-from coincurve.utils import verify_signature
+import ctypes
+import os
+import platform
+import warnings
+
+from coincurve._secp256k1_library_info import SECP256K1_LIBRARY_NAME, SECP256K1_LIBRARY_TYPE
```

Comment:
Please use lazy imports by moving all of these into the function and after the possible return statements.